src/components/_basic.less

```diff
@@ -44,6 +44,16 @@ a.sbtn {
   border-radius: 25px;
 }
 
+button:disabled {
+  opacity: 0.55;
+}
+
+a.btn-disabled {
```

Comment:
Please use `.sbtn.btn-disabled` instead of the general `a.btn-disabled`

Reply:
common class is added! Please review! thank you!